docs/Sprint-Review/sprint-27-summary.md

```diff
@@ -0,0 +1,62 @@
+
+
+## Submitted (QASP Review, OCIO Review)
+- Round 7 PreParsing Errors Live Comms w/ Regional Staff [#1194](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1194)
+- Round 7 (Sprint 24-26) UX Research and Design [#1017](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1017)
+
+## Moving to Next Sprint (Raft Review, In Progress, Current Sprint Backlog)
+**Raft Review**
+- As a dev, I'd like to be able to retain my database contents between restarts [#795](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/795)
+
+
+**Blocked**
+- As a dev, I want Terraform changes to be reflected in label driven deployments (GitHub Action) [#1059](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1059)
+
+**In Progress**
+- (Sprint 27) UX Research Support & Comms [#1166](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1166)
+- As a dev, I need to know which authentication service we're using (login.gov vs. NextGen XMS) [#638](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/638)
+- [Epic]Secret Key Leakage Mitigation [#972](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/972)
+- As a TDP user, I want to see a picture that better represents the program on the homepage [#640](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/640) 
+- Django Admin a11y Fixes (Sprint 3)  [#1054](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1054)
+- Round 7 PreParsing Errors Research Synthesis [#1193](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1193)
+- Remove "Super User" attribute, leverage model permissions [#901](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/901)
+- System Admin group by default [#1065](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1065)
+- Update/Create Target and Aspirational State Journey Maps Based on New Research and Roadmap Discovery [#1196](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1196)
+- [DevOps] Add a Circle CI testing step that confirms no secret keys have been committed [#966](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/966)
+- As a dev, I want the authentication credentials used for testing to be dynamically generated [#969](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/969)
```

Comment:
should this be moved to "blocked"?

Reply:
I added it to in progress b/c Andrew and John are actively working to unlock it today. I can move it if you like. LMK.

Reply:
That's fine to keep in progress if it's unblocked, we'll just want to update in ZenHub too